packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -20,19 +20,22 @@ import {
 } from '@carbon/react';
 import { useTranslation } from 'react-i18next';
 import {
-  useLayoutType,
+  ConfigurableLink,
+  ErrorState,
+  ExtensionSlot,
   isDesktop,
+  type OpenmrsResource,
   useConfig,
+  useLayoutType,
   usePagination,
-  ExtensionSlot,
-  ErrorState,
-  ConfigurableLink,
 } from '@openmrs/esm-framework';
 import { EmptyDataIllustration } from './empty-data-illustration.component';
-import { useActiveVisits } from './active-visits.resource';
+import { useActiveVisits, useActiveVisitsSorting, useObsConcepts } from './active-visits.resource';
 import styles from './active-visits.scss';
+import { type ActiveVisitsConfigSchema } from '../config-schema';
+import { type ActiveVisit } from '../types';
 
-function generateTableHeaders(t, config) {
+function generateTableHeaders(t, config, obsConcepts) {
```

Comment:
Move this function either in the component only, and remove the use the component state, or move this in the resource file.